tests/Doctrine/Tests/Common/Cache/MongoDBCacheTest.php

```diff
@@ -6,20 +6,34 @@
 use Doctrine\Common\Cache\MongoDBCache;
 use MongoClient;
 use MongoCollection;
+use MongoConnectionException;
 
 /**
  * @requires extension mongodb
  */
 class MongoDBCacheTest extends CacheTest
 {
+    private static $skip;
+
     /**
      * @var MongoCollection
      */
     private $collection;
 
     protected function setUp()
     {
-        $mongo = new MongoClient();
+        if (static::$skip === true) {
```

Comment:
`sleep()`??? 😭 

F*** that then, the `@requires` will not work because the connection is "lazy" so it fails only when trying to do something with it.

I'll keep the try/catch and remove that nasty static property... thanks